params/params.go

```diff
@@ -0,0 +1,26 @@
+package params
+
+import (
+	"github.com/juju/bundlechanges"
+	"github.com/juju/httprequest"
+)
+
+// ChangesResponse contains the results of parsing a bundle into a list of
+// changes.
+type ChangesResponse struct {
+	Changes []bundlechanges.Change
```

Comment:
It would be great if the bundle changes were serialised in exactly the same format as used by the juju API. Perhaps we could standardise a serialisation format and implement that in the bundlechanges package?